packages/app/non-translatable-markup/src/methods/isTextTranslatable.ts

```diff
@@ -1,10 +1,25 @@
-import { nonTranslatableTextRegexp } from './utils'
+import { nonTranslatableTextRegexp, symbolsRegexp } from './utils'
 
 /**
- * Returns true if the text is entirely encapsulated in non-translatable tags.
+ * Returns true if the text contain at least one translatable piece of text.
+ * A translatable piece of text is a piece of text that is not surrounded by non-translatable tags,
+ * symbols or a number.
  */
-export const isTextTranslatable = (text: string): boolean => {
-  const parts = text.split(nonTranslatableTextRegexp).map((part) => part.trim())
+export const isTextTranslatable = (text: string): boolean =>
+  trimAndFilterEmpty(text.split(nonTranslatableTextRegexp)).some((piece) =>
+    isPieceTranslatable(piece),
+  )
 
-  return parts.some((part) => part !== '')
+const isPieceTranslatable = (piece: string): boolean => {
+  // numbers are not translatable
+  if (!Number.isNaN(+piece)) return false
```

Comment:
Maybe for simplicity we could include numbers in regex pattern?

Reply:
Good point! done, thank youuu